### ISSUE TITLE
Fine tune "paraphrase-multilingual-MiniLM-L12-v2" on my domain data

### ISSUE THREAD

**Issue by parnab** [UKPLab/sentence-transformers#2303]
I have parallel language data in src_lang and target_lang in a tsv file like as below.
english \t korean
sent_en \t   sent_ko 
sent_en \t   sent_ko 

using the pretrained model  "paraphrase-multilingual-MiniLM-L12-v2" embeddings are fairly fine for most of the sentences between english and korean. However, it fails for some as well. Those are a little domain specific as we understand.

I was looking for a way to extend the model "paraphrase-multilingual-MiniLM-L12-v2" to fine tune on my parallel dataset so that embeddings similar for domain specific as well.

Can i get some guidance on how to initialize, format the training data and the loss to finetune the model

**Comment by aflip** [comment]
you might find[ this talk by  James Briggs](https://haystackconf.com/files/slides/haystack2022/James-Briggs-Training-Embedding-Models.pdf)   helpful in figuring out how to approach this. Additionally, sentence transformers website has a page on how to train and finetune your model. https://www.sbert.net/docs/training/overview.html

**Comment by tomaarsen** [comment]
Hello!

This page on [training multilingual models](https://www.sbert.net/examples/training/multilingual/README.html#training) should be helpful (although the TED2020 link is dead). Additionally, this script might help: https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/multilingual/make_multilingual.py

- Tom Aarsen
